src/caselawclient/xquery_type_dicts.py

```diff
@@ -6,72 +6,76 @@
 checks. They are used to enforce appropriately typed variables being passed in to MarkLogic XQuery functions.
 """
 
-from typing import Any, Optional, TypedDict
+from typing import Any, NewType, Optional, TypedDict
 
+MarkLogicDocumentURIString = NewType("MarkLogicDocumentURIString", str)
```

Comment:
Is this the right place for these definitions to live, or do we want a apiclient.typing module?

Reply:
It's not my favourite place for them – at first they lived in `Client` but this led to some circular import pain during testing, so I just moved them closer to where they're actually used.

We could bikeshed about this forever, and I'm fairly confident that a more clear place for things like this to exist will become apparent as we start doing more of properly nailing down the last few remaining bits of spit-and-sticky-tape stuff.